### ISSUE TITLE
Cannot install shellcheck via brew

### ISSUE THREAD

**Issue by qgaye** [koalaman/shellcheck#2229]
when I run `brew install shellcheck` the program throw ERROR `Failed to download resource "shellcheck_bottle_manifest"`

![image](https://user-images.githubusercontent.com/33546916/117671351-48119c80-b1db-11eb-90c1-513c33ddeb7d.png)

if I directly visit `https://ghcr.io/v2/homebrew/core/shellcheck/manifests/0.7.1-1` it show UNAUTHORIZED

![image](https://user-images.githubusercontent.com/33546916/117671502-6e373c80-b1db-11eb-9128-e4ce40ea3311.png)

**Comment by GhostLyrics** [comment]
@qgaye Update homebrew and try again, the latest shellcheck version available as of this comment is `0.7.2`.

**Comment by shinokada** [comment]
I have the same problem. I updated the brew.

```
$ brew install shellcheck
Error: shellcheck: no bottle available!
You can try to install from source with:
  brew install --build-from-source shellcheck
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.
```

**Comment by KeepmovingBenLee** [comment]
@shinokada I have the same problem as you have and I haven't fixed it.

**Comment by shinokada** [comment]
[ghc 8.8 may be a lost cause for Apple Silicon.](https://github.com/Homebrew/homebrew-core/pull/57892#issuecomment-735851664)

GHC is a Haskell and when I tried `brew install cabal-install`, it has the same problem.

![image](https://user-images.githubusercontent.com/147320/119481012-33add200-bd8d-11eb-8b5b-fa2e38c5b422.png)

**Comment by GhostLyrics** [comment]
Are you on a M1 based mac?

**Comment by shinokada** [comment]
Yes.

**Comment by shinokada** [comment]
It's not ideal but thanks for having an online shellcheck. I use https://www.shellcheck.net/.
I just wait until it's updated.
I don't recommend the M1 chip yet, because a quite few important apps are not working, like Docker.

**Comment by JackonYang** [comment]
@GhostLyrics's solution works for me, I'm using M1 chip.

here is the output:

<img width="1388" alt="Screen Shot 2021-06-05 at 11 46 04" src="https://user-images.githubusercontent.com/3008778/120879175-20ec9600-c5f4-11eb-9665-c2a88a73b53e.png">

what did I do

run `brew update` as below.
<img width="822" alt="Screen Shot 2021-06-05 at 11 50 51" src="https://user-images.githubusercontent.com/3008778/120879201-4c6f8080-c5f4-11eb-95e8-4ffcb05e0e27.png">

there is no output for a while, I killed the process and run `brew install shellcheck`. it works

**Comment by shinokada** [comment]
@JackonYang 
In your screenshot you have Errors. I don't think you installed it.
I updated and upgraded brew, but I still get an error.

![image](https://user-images.githubusercontent.com/147320/120879835-33210100-c601-11eb-9626-f1f230375f45.png)

**Comment by JackonYang** [comment]
@shinokada 

it works as you can see. there might be something wrong with your environment.

<img width="916" alt="Screen Shot 2021-06-05 at 13 18 21" src="https://user-images.githubusercontent.com/3008778/120881014-9f4f3500-c600-11eb-8326-98e1141623c6.png">

**Comment by shinokada** [comment]
I got the mac two weeks ago.
Are these OS and version the same as yours?

![image](https://user-images.githubusercontent.com/147320/120881102-ad558380-c609-11eb-8ea0-b80bbcf00144.png)

**Comment by bhrutledge** [comment]
I think this might be a duplicate of #2162, because you've got an M1 MacBook, and shellcheck doesn't natively support Apple Silicon yet. I posted a workaround in https://github.com/koalaman/shellcheck/issues/2162#issuecomment-855405285.

**Comment by dasl-** [comment]
It works now installing shellcheck with homebrew on M1 without rosetta:
```
% brew install shellcheck
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> New Formulae
at-spi2-atk                       libmobi                           maturin                           mongocli                          sqlbench                          tmuxp
==> Updated Formulae
Updated 224 formulae.
==> Renamed Formulae
grakn -> typedb
==> New Casks
kubenav               landrop               macstroke             p4                    prisma-studio         suuntodm5             universal-battle      wolfram-player        yippy
==> Updated Casks
Updated 135 casks.
==> Deleted Casks
crypt                                                               gitbox                                                              touch-bar-pong

==> Downloading https://ghcr.io/v2/homebrew/core/shellcheck/manifests/0.7.2
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/shellcheck/blobs/sha256:8844cd2a84bd07f8697463d9af3cc41cc4356edfa11d1171255884dc732ee9cd
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:8844cd2a84bd07f8697463d9af3cc41cc4356edfa11d1171255884dc732ee9cd?se=2021-06-10T16%3A15%3A00Z&sig=bjs71%2F1JETvtvGbD21D1
######################################################################## 100.0%
==> Pouring shellcheck--0.7.2.arm64_big_sur.bottle.tar.gz
🍺  /opt/homebrew/Cellar/shellcheck/0.7.2: 7 files, 68.8MB
```

**Comment by shinokada** [comment]
@dasl- Thanks. It works.

**Comment by qgaye** [comment]
> @qgaye Update homebrew and try again, the latest shellcheck version available as of this comment is `0.7.2`.

after update homwbrew, it works.
